src/ansys/fluent/core/services/batch_ops.py

```diff
@@ -120,9 +120,12 @@ def __init__(
                             ):
                                 self._supported = True
                                 response_cls_name = method_desc.output_type.name
-                                # TODO Get the respnse_cls from message_factory
-                                self.response_cls = getattr(file, response_cls_name)
-                                break
+                                # TODO Get the response_cls from message_factory
+                                try:
+                                    self.response_cls = getattr(file, response_cls_name)
```

Comment:
@hpohekar @prmukherj This will fix batch_ops test failure seen in some branch. The failure is seen in github only - added this defensive check as I couldn't find any issue with the code while investigating locally.

Reply:
Yes, it is failing in some branches. Thanks.